context-free.md

```diff
@@ -2,21 +2,28 @@
 
 ##  Who is this programming language for?
 
+Anyone who is interested in creating procedurally generated art.
 
 ## What is easy to do in this language? Why is it easy?
 
+Creating self similar images is very straight forward due to the way the rules call each other.
 
 ## What is hard to do in this language? Why is it hard?
 
+Controlling randomness seems a bit hard. It would be nice to do something other than just
+trying a bunch of different seeds to see what works.
 
 ## How did you learn how to program in this language?
 
+By looking at examples, both online and other students' submissions.
```

Comment:
Same. The documentation is useful too (although it's meant for the newer version of ContextFree)